UTRA-ART/Caffeine
# CAFF-76 - Increased local cost map size and adjusted sensor parameters

closes #76
Changes:
1. More explicit definitions and configurations of the obstacle and inflation layers were made for the local and global cost maps.
2. The size of the local cost map was increased to allow the point cloud from the ZED camera to register in the local cost map.
3. Both the laser and camera now contribute to the cost map when traversing the way points through no-man's-land.

How to test:
1. Caffeine's starting coordinates and orientation may need to be adjusted to the start of no-man's land (path: description -> launch -> spawn.launch).
2. Run standard gazebo, rviz, and move_base launch commands.
3. Run "roslaunch load_waypoints load_waypoints.launch" to traverse through no-man's land.

Justification:
The problem relating to the sensors' lack of contribution to the cost map is fixed in this pull request. There might still be some issues with the ZED camera and global cost map that still creates errors but this should be explored in another issue.
